_docs/portal/1.206.0.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: 1.206.0
+permalink: /docs/portal/1.206.0/
+parent: Portal
+nav_order: -204
+release_date: 07.10.2024
+---
+
+## Gates tab moved to all devices view
+The Gates tab has been relocated to the All Devices list, retaining all its features. Additionally, the Device Type filter allows you to display only gates within the All Devices list, effectively replicating the previous Gates list in a single view.
```

Comment:
Why =Gates, All Devices, Device Type starts with uppercase? I got that this is name on the device, but you never write 'I put beer into my Fridge'

Reply:
Just to highlight that it is "custom name" and to catch readers eye on it to be sure where it happens